torchinductor/scheduler.py

```diff
@@ -382,6 +383,78 @@ def get_priority(self):
             return len(self.group) - 1
 
 
+class FusedSchedulerNode(BaseSchedulerNode):
+    def __init__(self, scheduler: "Scheduler", snodes):
+        self.snodes = snodes
+        self.scheduler = scheduler
+        self.node = None
+        self.users = None
+        self.inverse_users = []
+        node = snodes[0].node
+        self.group = snodes[0].group
+        self.unmet_dependencies = self.snodes[0].read_writes.reads
```

Comment:
```suggestion
        self.unmet_dependencies = set(self.snodes[0].read_writes.reads)
```
Create a copy since you are mutating it below.  We want to leave the original one intact.